Assets/_BForBoss/_Weapons/Scripts/Weapons/WeaponBehaviour.cs

```diff
@@ -157,17 +147,9 @@ private void Awake()
 
         private void OnEnable()
         {
-            SetupPlayerInput();
             SetCrosshairImage();
         }
 
-        private void OnDisable()
-        {
-            _fireInputAction.started -= OnFireInputAction;
-            _fireInputAction.canceled -= OnFireInputAction;
-            _reloadInputAction.started -= OnReloadInputAction;
```

Comment:
if this is being removed, then these events are not being unsubscribed anywhere right?

Reply:
I Deinit it in the PGInputSystem